man/subscription-manager.8

```diff
@@ -29,7 +29,7 @@ Subscription Manager performs several key operations:
 .IP
 * It lists both available and used subscriptions.
 .IP
-* It allows administrators to both attach specific subscriptions to a system and remove those subscriptions.
+* It allows administrators to attach specific subscriptions to a system.
```

Comment:
this while point/line is moot now; there are further changes planned to this man page (see #3447), so I'd drop this change

Reply:
This is resolved by commit: [c42fd22](https://github.com/candlepin/subscription-manager/pull/3444/commits/c42fd2264ec7b4607c469a307232589cebc7b075)